packages/client/index.js

```diff
@@ -93,7 +94,12 @@ function hasDuplicatedParameters (methodMeta) {
   })
   return s.size !== methodMeta.parameters.length
 }
-function buildCallFunction (baseUrl, path, method, methodMeta, throwOnError, openTelemetry, fullRequest, fullResponse) {
+function buildCallFunction (spec, baseUrl, path, method, methodMeta, throwOnError, openTelemetry, fullRequest, fullResponse, validateResponse) {
+  // Resolve #ref(s) in schema, returns object { spec: THE_RESOLVED_SCHEMA }
+  const resolvedSchema = SwaggerClient.resolve({ spec })
+  const ajv = new Ajv({
+    strict: false // to avoid error on 'unknown keyword: $$ref' added by SwaggerClient
+  })
```

Comment:
I think somebody might want to customize those options.

Reply:
If we don't use `swagger-client` this is actually not needed